dashboard/app/controllers/api/v1/schools_controller.rb

```diff
@@ -0,0 +1,7 @@
+class Api::V1::SchoolsController < ApplicationController
+  # GET /api/v1/school/<school_district_id>/<school_type>
+  def index
+    schools = School.where(school_district_id: params[:school_district_id], school_type: params[:school_type])
```

Comment:
http://stackoverflow.com/questions/21886170/best-way-to-go-about-sanitizing-user-input-in-rails suggests this isn't necessary because we are using the `hash` form of `where`. Please let me know if you disagree.